data-raw/default_parameter_ae.R

```diff
@@ -25,6 +25,28 @@ default_parameter_ae <- list(
     summ_row = "with drug-related{^a} adverse events",
     summ_foot = "{^a}Determined by the investigator to be related to the drug."
   ),
+  g34 = adam_mapping(
+    name = "g34",
+    label = "grade 3-4 adverse events",
+    subset = quote(ATOXGRN %in% c(3, 4)),
+    var = "AEDECOD",
+    soc = "AEBODSYS",
+    seq = 250,
```

Comment:
Hi Ruchit, may I know what the `seq = ...` means?

Reply:
It controls the order to display rows in the final table as we will sort by `seq`.